### ISSUE TITLE
[RFC][Semantic-ui] Use semantic less instead of semantic-ui-css

### ISSUE THREAD

**Issue by Niiko** [Sylius/Sylius#5822]
I think it would be better to use the less version of semantic-ui to ease overriding of sylius-theme.

In that way, we only have to add a line in gulpfile to override variables instead of completely modify gulpfile.js in our projects.

WDYT ?

**Comment by inssein** [comment]
I actually set this up the other day - it takes a bit if work to get it right, and adds a lot of files to the project, but it is probably worth it.

My only concern for Sylius would be that it sort of messes with the concept of themes in the ThemeBundle because the ThemeBundle has a concept of themes, and then Semantic has its own concept of themes.

**Comment by Niiko** [comment]
Yes, you are right, there is some inconsistency between files from gulpfile.js (css, js, sass) and ThemeBundle files that could include itself some files.

In fact, i have my own theme using ThemeBundle, and i load my specific css files for the theme in Gulpfile.

Maybe we can have in gulpfile a parameter with the theme directory used, default or custom, and load that files automatically.

Even better, take that parameter from the theme configured, but i think it's kinda hard to access it from gulp, no ?

**Comment by inssein** [comment]
Yeah, I thought about a workaround which involved defining the theme in configuration, but that limits ThemeBundle as it can have a different theme per request.

**Comment by Niiko** [comment]
Right, so it has to generate files for each theme installed instead.

We could have a directory like this

```yaml
assets/
   /shop
        /default
              /css
              /img
              /js
        /theme1
              /css
              /img
              /js
        /theme2
              /css
              /img
              /js
```

**Comment by teohhanhui** [comment]
Hmm... I believe this is still an issue.

**Comment by jacquesbh** [comment]
Actually it could be great of being able to disable Semantic UI and having different processing for admin and frontend designs. This way we could keep Semantic UI in admin and move on to something else (with themes) on frontend.

**Comment by alekseyp** [comment]
https://medium.com/@naomiHauret/i-used-semantic-ui-as-a-css-framework-and-regret-my-choice-aa4449ab6107 - found this while googling
